composer.json

```diff
@@ -20,7 +20,8 @@
         "google/recaptcha": "^1.2",
         "pragmarx/google2fa-laravel": "^1.3",
         "spatie/laravel-schemaless-attributes": "^1.8",
-        "spatie/laravel-flash": "^1.7"
+        "spatie/laravel-flash": "^1.7",
+        "php": "^7.4|^8.0"
```

Comment:
Should be the first entry in the requirements.

Reply:
Fixed.